tests/integration/features/conversation/add-group.feature

```diff
@@ -0,0 +1,212 @@
+Feature: public
+  Background:
+    Given user "participant1" exists
+    Given user "participant2" exists
+    Given user "participant3" exists
+    And group "group1" exists
+    And user "participant2" is member of group "group1"
+
+  Scenario: Owner invites a group
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 3 |
+      | roomName | room |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+    And user "participant1" adds group "group1" to room "room" with 200 (v4)
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+
+  Scenario: Owner start a chat with a group
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2 |
+      | source | group |
+      | invite |group1 |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+
+  Scenario: User is added to a group which is a member of a chat
+    Given user "participant1" creates room "room" (v4)
+      | roomType | 2 |
+      | source | group |
+      | invite |group1 |
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+    And user "participant3" is member of group "group1"
+    And user "participant1" sees the following attendees in room "room" with 200 (v4)
+      | actorType  | actorId      | participantType |
+      | users      | participant1 | 1               |
+      | groups     | group1       | 3               |
+      | users      | participant2 | 3               |
+      | users      | participant3 | 3               |
+
+  Scenario: User is removed from a group which is a member of a chat
```

Comment:
I think that it would be good to have scenarios too for removing a user from a single group when the user belongs to two different groups that are members of a conversation, and removing just one of the groups that are members of a conversation.